src/v/features/feature_table.h

```diff
@@ -99,7 +99,8 @@ inline const std::unordered_set<std::string_view> retired_features = {
   "mtls_authentication",
   "rm_stm_kafka_cache",
   "transaction_ga",
-};
+  "idempotency_v2",
+  "transaction_partitioning"};
```

Comment:
how do you make sure the feature flag is not in use anymore?

Reply:
https://redpandadata.atlassian.net/wiki/spaces/CORE/pages/170491930/Features+Cluster+Logical+Versioning is relevant. Once the cluster is fully upgraded, the feature is always true and this coupled with no rolling back requirement across minors guarantees it is always true.

Reply:
so how do you know all clusters are upgraded so that the feature is always true on all of them?